dbms/src/Storages/Page/V3/tests/gtest_page_directory.cpp

```diff
@@ -1960,27 +1960,27 @@ TEST_F(PageDirectoryGCTest, RestoreWithRef)
 try
 {
     BlobFileId file_id1 = 1;
-    BlobFileId file_id2 = 5;
+    BlobFileId file_id2 = 2;
 
     const auto & path = getTemporaryPath();
     createIfNotExist(path);
     Poco::File(fmt::format("{}/{}{}", path, BlobFile::BLOB_PREFIX_NAME, file_id1)).createFile();
     Poco::File(fmt::format("{}/{}{}", path, BlobFile::BLOB_PREFIX_NAME, file_id2)).createFile();
 
     PageEntryV3 entry_1_v1{.file_id = file_id1, .size = 7890, .tag = 0, .offset = 0x123, .checksum = 0x4567};
-    PageEntryV3 entry_5_v1{.file_id = file_id2, .size = 255, .tag = 0, .offset = 0x100, .checksum = 0x4567};
-    PageEntryV3 entry_5_v2{.file_id = file_id2, .size = 255, .tag = 0, .offset = 0x400, .checksum = 0x4567};
+    PageEntryV3 entry_2_v1{.file_id = file_id2, .size = 255, .tag = 0, .offset = 0x100, .checksum = 0x4567};
+    PageEntryV3 entry_2_v2{.file_id = file_id2, .size = 255, .tag = 0, .offset = 0x400, .checksum = 0x4567};
     {
         PageEntriesEdit edit;
-        edit.put(file_id1, entry_1_v1);
-        edit.put(file_id2, entry_5_v1);
+        edit.put(1, entry_1_v1);
+        edit.put(5, entry_2_v1);
```

Comment:
revert back these file id changes?

Reply:
revert the `entry_2_v1 ` to `entry_5_v1 `, but need put `page_id` not the `file_id` here , so i changed `file_id` to number